wfe/wfe.go

```diff
@@ -1386,45 +1406,48 @@ func (wfe *WebFrontEndImpl) deactivateAuthorization(ctx context.Context, authz *
 	return true
 }
 
-// Authorization is used by clients to submit an update to one of their
-// authorizations.
-func (wfe *WebFrontEndImpl) Authorization(ctx context.Context, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) {
+// AuthorizationV2 is used by clients to submit an update to an authzv2-style
+// authorization.
+func (wfe *WebFrontEndImpl) AuthorizationV2(ctx context.Context, logEvent *web.RequestEvent, response http.ResponseWriter, request *http.Request) {
 	// Requests to this handler should have a path that leads to a known authz
 	id := request.URL.Path
 	var authz core.Authorization
 	var err error
-	if features.Enabled(features.NewAuthorizationSchema) && strings.HasPrefix(id, authz2Prefix) {
-		authzID, err := strconv.ParseInt(strings.TrimPrefix(id, authz2Prefix+"/"), 10, 64)
-		if err != nil {
+	if !features.Enabled(features.NewAuthorizationSchema) {
+		wfe.sendError(response, logEvent, probs.NotFound("No such authorization"), nil)
```

Comment:
nit: elsewhere you have a `notFound` inline function that would write the "No such xxx" error. I think that pattern would make sense here too since the "No such authorization" error is written ~3 places.